intuit.css

```diff
@@ -0,0 +1,163 @@
+.body_style {
+  background-color: #dfdfdf;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.sign-in-part {
+  width: 23%;
+  float: right;
+  display: block;
+  overflow: hidden;
+  margin: 10px;
+}
+
+.center {
+  margin: auto;
+  width: 40%;
+  padding: 40px;
+  height: auto;
+}
+
+.centerbase {
+  margin: auto;
+  padding-top: 100px;
+  font-size: smaller;
+  width: 55%;
+  padding-bottom: 20px;
+}
+
+.top_image {
+  padding-bottom: -20px;
+  padding-left: 40px;
+}
```

Comment:
Remove these two properties please 
```css
padding-bottom: -20px;
padding-left: 40px;
```

Try something like
```css
.top_image {
  display: flex;
  margin: auto;
  justify-content: space-around;
}
```

Reply:
changed the top_image class property as per the recommendations